css/about.css

```diff
@@ -0,0 +1,167 @@
+/* I tried my best but couldn't go far enough to replicate pixel-for-pixel design */
+
+
+/* My code starts here */
+
+.navigation {
+    margin: 5px;
+    width: 100%;
+    text-align: right;
+    font-size: 20px;
+    padding: 20px;
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.container {
+    width: 1000px;
+    margin: 0px auto;
+}
+
+p {
+    padding: 10px auto;
+    font-size: 18px;
+}
+
+h2 {
+    padding: 5px;
+}
+
+h3 {
+    padding: 5px;
+}
+
+.logo {
+    margin: 15px;
+    position: absolute;
+    padding: 10px;
+    left: 0;
+    /* top: 10; */
+    /* display:  */
+}
+
+.jumbo-about {
+    width: 1000px;
+    padding: 10px;
+    display: flex;
+    justify-content: center;
+}
+
+button {
+    border: 1px solid gray;
+    padding: 10px;
+    font-size: 10px;
+    margin: 20px auto;
+    display: inline;
+    cursor: pointer;
+}
+
+.about-top {
+    margin: 10px auto;
+    padding: 5px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+
+.strategy-top {
+    /* border: 1px solid red; */
+    /* border: 1px solid gray; */
+    margin: 20px 10px;
+    /* line-height: 1.25; */
+    padding: 10px 10px;
+}
+
+.strategy {
```

Comment:
Like I mentioned before, these classes should go on the h3 tags instead of the section tags. The CSS is great though! Great work!